### ISSUE TITLE
Bookmark request an some more sources

### ISSUE THREAD

**Issue by Recoil1024** [zyrouge/yukino-app#177]
Man pls add bookmark feature and also add more sources for manga and anime like before it had 5 something of sources for manga and anime

**Comment by joex92** [comment]
I suggest some spanish subs anime sites:

- https://www3.animeflv.net/
- https://sololatino.net/animes/
- https://animeyt.es/

**Comment by zyrouge** [comment]
> I suggest some spanish subs anime sites:
> 
> * https://www3.animeflv.net/
> * https://sololatino.net/animes/
> * https://animeyt.es/

Please only request extension [here](https://github.com/yukino-app/extensions-store/issues/19), and I don't plan on making & maintaining non-english extensions as I have to maintain the whole app ecosystem all by myself. Contributions are always welcomed.

**Comment by joex92** [comment]
oh ok, I understand... is there an extension API or something like that?

**Comment by zyrouge** [comment]
> oh ok, I understand... is there an extension API or something like that?

Yes, there is an API but you should wait since I'm making some breaking changes.
